tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -310,10 +317,18 @@ def fit(train_data,
             where `L` ranges from 0 to 50 (Note: higher values of `L` correspond to fewer print statements, opposite of verbosity levels)
 
         Kwargs can include additional arguments for advanced users:
+            AG_args : dict, default={}
+                Keyword arguments to pass to all models. See the `AG_args` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
```

Comment:
"Keyword arguments to pass to all models."
->
"Keyword arguments to pass to all models (i.e. common hyperparameters shared by all AutoGluon models)."

Reply:
Added